tools/amd_build/disabled_features.yaml

```diff
@@ -200,6 +225,13 @@
       "functions": [
         "THCTensor_(getTextureObject)"
       ]
+    },
+    {
+      "path": "aten/src/THC/THCTensorRandom.cu",
+      "functions": [
+        "THCRandom_setRNGState",
+        "set_rngstate_kernel"
```

Comment:
For set_rngstate_kernel, [until this is handled ](https://github.com/ROCmSoftwarePlatform/rocRAND/issues/22), you can manually pass in the fields like

```state[threadIdx.x].pos_tbl = kernel.pos_tbl;
state[threadIdx.x].param_tbl = kernel.param_tbl;
state[threadIdx.x].temper_tbl = kernel.temper_tbl;
state[threadIdx.x].single_temper_tbl = kernel.single_temper_tbl;
state[threadIdx.x].sh1_tbl = kernel.sh1_tbl;
state[threadIdx.x].sh2_tbl = kernel.sh2_tbl;
state[threadIdx.x].mask = kernel.mask;```

Reply:
I'd like to avoid patch files however (or an `#ifdef`, I see no other options). So unless is function is crucial, maybe we can wait till that is fixed upstream?

Reply:
You won't be able to set the random number generator state for the GPU without it.